test-harness/specs/default_responses-AC-5.oas2.txt

```diff
@@ -0,0 +1,26 @@
+====test====
+When I send a request to an operation
+And in the path I specify ?_code=some-code (ie ?_code=200)
+And the operation has such status code response defined
+Then I get back a response with a status code of 200
+====spec====
+swagger: "2.0"
+paths:
+  /todos:
+    get:
+      responses:
+        200:
+          description: ok
+          schema:
+            type: string
+            example: ok
+====server====
+mock -p 4010
+====command====
+curl -i -X GET http://localhost:4010/todos?_code=200
+====expect====
+HTTP/1.1 200 OK
+content-type: */*
```

Comment:
@philsturgeon Same here, this needs an investigation

Reply:
@philsturgeon I'm struggling here. What would be the content type for this response?